lib/server/handlers/sthRemoveDataHandler.js

```diff
@@ -64,7 +64,10 @@ function removeDataHandler(request, reply) {
         },
         function(err) {
             if (err) {
-                if (err.name === 'MongoError' && err.message.indexOf('does not exist. Currently in strict mode')) {
+                if(err.name === 'MongoNetworkError' && err.message.includes('failed to connect to server')){
```

Comment:
Aligned with https://github.com/telefonicaid/fiware-sth-comet/pull/583/files#r813167242, maybe just:

```suggestion
                if(err.name === 'Not connected')){
```